src/index.js

```diff
@@ -95,7 +95,7 @@ async function run() {
 					let cp_options    = ( is_dir ) ? { recursive: true, force: true } : {},
 						iscopied      = true,
 						dest_basepath = toolkit.path.dirname( `${local_path}${workflow_file.dest}` ),
-						copy_source   = ( is_dir ) ? `${toolkit.path.trailingslashit( source_path )}.` : source_path;
```

Comment:
i added this line to make sure that i can copy files / folder if folder & file exists with the same name.

So it would be nice if you can create a workaround by adding if conditions

Reply:
Sorry, I am not a javascript person. You are welcome to use this code and branch to another PR! I am unclear how to solve #2 with the `\.` without removing this